extensions/jellyfin/README.md

```diff
@@ -0,0 +1,58 @@
+# Jellyfin Extension for Raycast
+
+This project focuses on developing an extension for the application "Raycast".
```

Comment:
This line is not really necessary, as it will show on our repo and on our store.